<launch>
    <param name="dh_a2" type="double" value="0.27"/>
    <param name="dh_a3" type="double" value="0.07"/>
    <param name="dh_d1" type="double" value="0.29"/>
    <param name="dh_d4" type="double" value="0.302"/>
    <param name="dh_a6" type="double" value="0.046"/>
    <param name="dh_dT" type="double" value="0.2069"/>
</launch>